pkgs/applications/editors/vim/plugins/deprecated.json

```diff
@@ -12,7 +12,7 @@
         "new": "vim-fern"
     },
     "gina-vim": {
-        "date": "2024-06-19",
```

Comment:
(Not familiar with vimPlugins, just asking) Why do we have to change date in deprecated plugins?

Reply:
I find the warning useful but it's strange that the date gets updated ?!

Reply:
The update script matches it, not excluding it from the path?